modules/bitgo/src/v2/coins/tavaxp.ts

```diff
@@ -0,0 +1,31 @@
+import { BitGo } from '../../bitgo';
+import { BaseCoin } from '../baseCoin';
+import { AvaxP } from './avaxp';
+import { BaseCoin as StaticsBaseCoin } from '@bitgo/statics';
+
+export class TavaxP extends AvaxP {
```

Comment:
```suggestion
export class TAvaxP extends AvaxP {
```

Reply:
I think we should follow the syntax of avaxc and keep things consistent.  TavaxC is how its written.